docs/meshstack.aws.aad-as-idp.md

```diff
@@ -0,0 +1,80 @@
+---
+id: meshstack.aws.aad-as-idp
+title: AAD as Identity Provider
+---
+## Configure AAD as IdP for AWS SSO
+
+You can trigger everything that is needed for using AAD as an IdP for AWS SSO from within Azure.
+
+### Create an Enterprise Registration
+
+Search for `aws` and choose `AWS Single Sign-on` as the enterprise registration type
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png)
+
+Follow the steps in AAD under Getting Started (Pick SAML for the single sign on.)
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png)
+
+### Configure AWS for AAD integration
+
+Choose `AWS Single Sign-on`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_03.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_03.png)
+
+
+Configure the identity source by clicking on `Choose your identity source`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_04.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_04.png)
+
+We have to change the `Identity source`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_05.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_05.png)
+
+Select `External identity provider`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_06.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_06.png)
+
+### Metadata
+
+Download `AWS SSO SAML metadata` from AWS and then you should upload it into `Single sing-on` page on Azure with the `Upload metadata file` button.
```

Comment:
```suggestion
Download `AWS SSO SAML metadata` from AWS and then you should upload it into `Single Sign-On` page on Azure with the `Upload metadata file` button.
```

Reply:
hehe, but sing-on is also nice, lalalala :D